### PR TITLE
first steps for frontend

### PR DESCRIPTION
**Pull request by jpalmieri**
@sfjuggernaut 

I originally did a bunch of front end stuff in this branch: https://github.com/keplerites/kepler/tree/cover-slide-stars

But, despite it's superficial beauty, it's really ugly under the hood. This PR intends to lay the groundwork for reproducing that branch in a more rails-y way.

I started out by adding the resources for performers, team members, and musicians. I then added the [parallax.js](https://github.com/wagerfield/parallax) star background. To make the star background smoooooth and not jerk every time a link is clicked (because the whole body is being refreshed), I used [pjax](https://github.com/eval/rack-pjax) (the rack version; the rails one seems to be rarely maintained) in favor of turbolinks to swap out the html in the container which wraps the `yield` in the layout. From my understanding, pjax basically does the same thing as turbolinks, but offers more customization.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -33,6 +30,13 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+# javascript libraries
+gem 'jquery-rails', '~> 4.0'
+gem 'rack-pjax'
```
Comment (sfjuggernaut) [inline Gemfile]: Should we specify a version for this gem?

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -10,6 +10,14 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
+//= require jquery3
+//= require jquery_ujs
+//= require jquery.pjax
 //= require rails-ujs
-//= require turbolinks
+//= require parallax.min.js
```
Comment (sfjuggernaut) [inline app/assets/javascripts/application.js]: So weird that the asset pipeline uses this style - cause it looks like everything is commented out but 👍

File: app/controllers/music_controller.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+class MusicController < ApplicationController
+  def index
+    @musicians = CONTENT[:music]
+  end
+
+  def show
+    musicians = CONTENT[:music]
+    musician = musicians.select do |x|
+      x[:simple_name] == params[:id]
+    end
```
Comment (sfjuggernaut) [inline app/controllers/music_controller.rb]: Slightly tighter:

```
musician = musicians.select {|m| m[:simple_name] == params[:id]}.first()
```

File: app/controllers/music_controller.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+class MusicController < ApplicationController
+  def index
+    @musicians = CONTENT[:music]
+  end
+
+  def show
+    musicians = CONTENT[:music]
+    musician = musicians.select do |x|
```
Comment (sfjuggernaut) [inline app/controllers/music_controller.rb]: Slightly tighter:

```
musician = musicians.select {|m| m[:simple_name] == params[:id]}.first()
```

**Review by sfjuggernaut — APPROVED** [review]
Didn't look over the css / js changes too intensely, but other than small nit, these changes look 👍 

And yeah I like your idea of merging in the structural changes before the biggie PR.

File: app/assets/javascripts/application.js (hunk 2 above)
Comment (jpalmieri) [reply app/assets/javascripts/application.js]: yeah, I never really understood that either ¯\_(ツ)_/¯

File: app/controllers/music_controller.rb (hunk 4 above)
Comment (jpalmieri) [reply app/controllers/music_controller.rb]: 👍

File: Gemfile (hunk 1 above)
Comment (jpalmieri) [reply Gemfile]: oooo good point. I was moving too fast! 🚀

**Comment by jpalmieri** [comment]
@sfjuggernaut thanks for looking this over. I'm going to leave this unmerged for now, since the css is still really barebones and there are some content changes that I'm waiting to get some wireframes for. I'll submit another PR for the styling and additional content, and have it compare to this branch.

**Pull request merged**